Sources/TuistCache/Utilities/CacheXCFrameworkBuilder.swift

```diff
@@ -63,6 +63,19 @@ public final class CacheXCFrameworkBuilder: CacheArtifactBuilding {
                 )
             }
 
+            // Build for the macCatalyst
+            var macCatalystArchivePath: AbsolutePath?
+            if platform == .iOS {
+                macCatalystArchivePath = temporaryDirectory.appending(component: "macCatalyst.xcarchive")
+                try await self.macCatalystBuild(
```

Comment:
Shouldn't this be done only if `isMacCatalystSupported`? Otherwise we are building it and then trhowing it away, isn't it?

Reply:
Thank you for the insightful feedback! As you pointed out, I have added the necessary branch to make use of the generated output only if `isMacCatalystSupported` is true.